Ansible/playbook.yml

```diff
@@ -0,0 +1,136 @@
+---
+- name: Security Hardening on Ubuntu
+  hosts: all
+  gather_facts: no
+  become: yes
+  vars:
+    ssh_public_keys:
```

Comment:
In my opinion, these variables can be better structured. I can see that `ssh_public_keys` are keys for the new_user.
Something like:
```yaml
new_user:
  name: ...
  password: ...
  ssh_public_keys: ...
```
And also ... Is the password really needed here?